questionary/prompts/password.py

```diff
@@ -55,7 +56,26 @@ def password(
     Returns:
         :class:`Question`: Question instance, ready to be prompted (using ``.ask()``).
     """
+    # Créer un validateur si nécessaire
+    validator = build_validator(validate)
 
-    return text.text(
-        message, default, validate, qmark, style, is_password=True, **kwargs
-    )
+    def ask():
```

Comment:
mhm I am not quite sure why this is needed, the `is_password=True` should already ensure that the password is not visible and replaced with `*`. Can you describe a bit more what the advantace advantage of getpass is?